### ISSUE TITLE
smp.lossess says error message "'FocalLoss' object has no attribute '__name__'"

### ISSUE THREAD

**Issue by Shima-shoki** [qubvel/segmentation_models.pytorch#623]
Hi, there seems to be an error in the "smp.losses" module.
Several loss functions are listed in the documentation webpage (https://smp.readthedocs.io/en/latest/losses.html#lovaszloss), but all of them say the same error massage that there is "no attribute '__name__'". The loss functions from "utils.losses" work fine.
The version of smp is "0.3.0".
"train_epoch" and "valid_epoch" are constructed from the "utils.train.TrainEpoch" and "utils.train.ValidEpoch", respectively.
The error messages are shown below. Thank you in advance for your help!

![Untitled](https://user-images.githubusercontent.com/47880765/182010130-8dae9034-c64e-45f1-bfef-9418445d0aec.png)

**Comment by Shima-shoki** [comment]
Sorry, the solution to this problem is in the closed issues...
I should have checked it thoroughly first before asking😅
However, I'm still not sure this is an intended behavior of the code or some kind of bugs that can be fixed in the future releases... :)

https://github.com/qubvel/segmentation_models.pytorch/issues/358#issue-821087812
